packages/editor/src/components/block-settings-menu/reusable-block-convert-button.js

```diff
@@ -49,25 +49,33 @@ export function ReusableBlockConvertButton( {
 
 export default compose( [
 	withSelect( ( select, { clientIds } ) => {
-		const { getBlock, getReusableBlock } = select( 'core/editor' );
+		const { getBlock, canInsertBlockType, getReusableBlock } = select( 'core/editor' );
 		const { getFallbackBlockName } = select( 'core/blocks' );
 
 		const blocks = map( clientIds, ( clientId ) => getBlock( clientId ) );
+		if ( some( blocks, ( block ) => ! block ) ) {
+			return { isVisible: false };
+		}
 
 		// Hide 'Add to Reusable Blocks' on Classic blocks. Showing it causes a
 		// confusing UX, because of its similarity to the 'Convert to Blocks' button.
-		const isVisible = (
-			every( blocks, ( block ) => !! block ) &&
```

Comment:
Just for my own understanding:
Do you know how it is possible for us to have a `clientId` that doesn't have a corresponding block?

Reply:
I played around with this a little and this check seems to prevent an unknown property exception from being thrown when a static block is converted into a reusable one. 

I added a comment in 4bab114 which explains that.

Still, I'm not clear on how this happens. `withSelect` must be getting called after the static block is removed but before the block list has a chance to update... 🤔